modules/system/models/MailLayout.php

```diff
@@ -152,6 +152,12 @@ public function fillFromView($path)
 
     protected static function getTemplateSections($code)
     {
-        return MailParser::parse(FileHelper::get(View::make($code)->getPath()));
+        try {
+            $view = View::make($code);
+            return MailParser::parse(FileHelper::get($view->getPath()));
+        }
+        catch (InvalidArgumentException $e) {
+            return null;
+        }
```

Comment:
Would probably be simpler to just check if the view exists first. Are the consumers of this method going to fail if they encounter null instead of an array?

Reply:
Yeah, I'm addressing this in next commit.